dynamics-nav/How-to--Install-the-Microsoft-Dynamics-NAV-Provisioning-Tools-for-Microsoft-Azure.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Install the Microsoft Dynamics NAV Provisioning Tools for Microsoft Azure"
+description: "This topic describes how to install the Microsoft Dynamics NAV Provisioning Tools for Microsoft Azure from files available in the WindowsPowerShellScripts folder on the Dynamics NAV installation media (DVD)."
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.